ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -96,18 +107,48 @@ codeAction lsp state _ (TextDocumentIdentifier uri) _range CodeActionContext{_di
           | x <- xs
           , Just ps <- [annotatedPS]
           , Just dynflags <- [df]
-          , (title, graft) <- suggestExactAction exportsMap dynflags ps x
-          , Right edit <-
-                [ -- either (Left . traceShow) Right $
-                 rewriteToEdit dynflags uri (annsA ps) graft
-                 ]
+          , (title, grafts) <- suggestExactAction exportsMap dynflags ps x
+          , let edit = foldMapBy unionWSEdit mempty (either error id .
+                        rewriteToEdit dynflags uri (annsA ps)) grafts
+          ] ++
+          [mkCA title [x] edit
```

Comment:
Isn't there any way to package `suggestImportDisambiguation` inside `suggestExactAction`? 
As seen in #1273 it is strongly preferable to for all the code action "providers" to have a compatible signature, so that they can be reordered easily

Reply:
I first did so, but some AST manipulation failed with ghc-exactprint so I decided to make it return `[Either TextEdit Rewrite]`. If we could rewrite the code solely by ghc-exactprint, I will be happy to package it into `suggestExactAction`, which was my first attempt.

Reply:
I finally reunite `suggestImportDisambiguation` into `suggestExactAction`. Thank you!